src/client/app/App.js

```diff
@@ -1,24 +1,20 @@
 import React, { Component } from 'react'
 // containers
-import AcctInput from './container/AcctInput.container'
-import Accts from './container/Accts.container'
-import TableOptions from './container/TableOptions.container'
-import DataTable from './container/DataTable.container'
+import AcctInputContainer from './containers/AcctInputContainer'
+import AcctsContainer from './containers/AcctsContainer'
+import TableOptionsContainer from './containers/TableOptionsContainer'
+import DataTableContainer from './containers/DataTableContainer'
```

Comment:
the `.container` thing is nice sometimes, but without Container in the class's name it's hard (from a person not familiar with the app) to tell what it can be at the bottom of the page without scrolling up to look at the imports